rocAL/rocAL_pybind/examples/rocAL_api_caffe2_reader.py

```diff
@@ -35,7 +35,7 @@ def draw_patches(img, idx):
         image = img.cpu().numpy()
     else:
         image = img.detach().numpy()
-    image = image.transpose([1, 2, 0])
+    # image = image.transpose([1, 2, 0])
```

Comment:
We need to handle both the NHWC & NCHW cases. Add that

Reply:
Make sure we dont have commented lines in the files

Reply:
ToT has the transpose for both cases. That's why we commented it out for the NHWC case but I'll add the case for NCHW also

Reply:
Only one case is handled in this example as tensor layout is not even passed anywhere to any augmentation

Reply:
I think we do pass the layout from args . Get it from args.NCHW / NHWC ans plot the images